docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,170 @@
+# Development
+
+This repository is set up to be built in GitHub Actions.
+See [`.github/workflows/build-packages.yml`](.github/workflows/build-packages.yml).
+
+On a successful build, the resulting packages are uploaded to https://anaconda.org/litex-hub/repo and can be installed
+with: `conda install --channel "LiteX-Hub" package`.
+
+These packages are mostly used by [`litex-buildenv`](https://github.com/litex-hub/litex-buildenv).
```

Comment:
I think this is not true anymore - they are being used in tons of places which are more up to date, so I'd drop the reference.

Reply:
Actually, I did not update the content (yet). I just moved it and put it in a site. Let's update it in a follow-up PR, along with adding refs to other conda-* repos.